naga/Cargo.toml

```diff
@@ -116,10 +116,9 @@ itertools.workspace = true
 ron.workspace = true
 rspirv.workspace = true
 serde = { workspace = true, features = ["default", "derive"] }
-spirv = { workspace = true, features = ["deserialize"] }
 strum = { workspace = true }
-toml.workspace = true
 walkdir.workspace = true
+wgpu-test.workspace = true
```

Comment:
I'm pretty uncomfortable making this back edge - I feel like there's a better design for sharing the benchmarks. More specifically I'm not really sure why this backedge needs to be here at all if the tests have been moved to wgpu-test.